Dixit_Logic/Interfaces/IImageAccess.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Drawing;
+
+namespace Dixit_Logic.Interfaces
+{
+    /// <summary>
+    /// This interface loads the images for the cards.
+    /// </summary>
+    interface IImageAccess
```

Comment:
Yes. As a general rule of thumb make everything public that you want others to access. 

Otherwise looks good.